microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -391,6 +420,33 @@ Kids Mode does not collect data for personalization of the news feed or other Mi
+For non-members, Microsoft Edge presents personalized joining offers based on the page you’re on. Once you engage and choose to enroll, Edge uses your Microsoft account identity to sign you up and start the rewards service that automatically adds points to your account for searching, playing and shopping on Edge. Edge does not send your personal information or browsing data to any third-party services. 
+Periodically, Edge could also offer custom browsing streak offers for just browsing on Edge for a few minutes every day for a limited period. You can track your streak progress; points earned and discover new offers on the [Rewards dashboard](https://rewards.bing.com/) or the action center on the Edge sidebar. 
+
+![RewardsDashboardUI](./media/rewardsdashboard.png)
+
+![RewardsSidepaneUI](./media/rewardssidepane.png)
+
+Rewards is an opt in program. You can choose to turn off all Rewards experiences and data sharing in the Edge settings at : edge://settings/profiles/rewards. 
+
+![RewardsSettingsUI](./media/rewardssettings.png)
+
+You can also completely opt out of the Rewards program. Please see the the details below: 
+Note:
+1.	Offers may vary by market. 
+2.	How to earn: Microsoft Edge provides the greatest number of avenues to earn rewards points while browsing the web. 
+a.	A search is the act of an individual user manually entering text for the good faith purpose of obtaining Bing search results for such user’s own research purposes and does not include any query entered by a bot, macro, or other automated or fraudulent means of any kind ("Search").
+b.	An acquisition is the process of purchasing goods or downloading and acquiring a license for digital content from Microsoft, whether free or paid ("Acquisition"). Rewards points are not offered for every purchase from Microsoft. Microsoft may offer additional opportunities to earn points from time to time, and each point-earning offer will not be available in perpetuity.
+3.	Device requirement: You need a valid Microsoft account, and your devices must meet the minimum [system requirements](https://account.microsoft.com/rewards/).
+4.	Redeeming your points: You may redeem your points, or you may contribute points to a listed nonprofit organization by visiting the [redeem page](https://aka.ms/redeemrewards) or by signing up for automatic contributions to non-profit organizations through a giving program.
+5.	Cancelling Your Participation: If you no longer wish to participate in the Microsoft Rewards program, follow the instructions on the [opt out page](https://account.microsoft.com/rewards/optout?confirm=false). If you opt out, you will immediately lose all of your available points. 
+6.	For more information see the Rewards section at support.microsoft.com ("FAQ") and refer to Rewards section in the [Microsoft service agreement](https://www.microsoft.com/en-us/servicesagreement/).
```

Comment:
```suggestion

Offers may vary by market. 
How to earn: Microsoft Edge provides the greatest number of avenues to earn rewards points while browsing the web. 
A _search_ is the act of an individual user manually entering text for the good faith purpose of obtaining Bing search results for the user's own research purposes, and does not include any query that's entered by a bot, macro, or other automated or fraudulent means of any kind ("Search").
An _acquisition_ is the process of purchasing goods or downloading and acquiring a license for digital content from Microsoft, whether free or paid ("Acquisition"). Rewards points are not offered for every purchase from Microsoft. Microsoft may offer additional opportunities to earn points from time to time, and each point-earning offer will not be available in perpetuity.
#### Device requirement
You need a valid Microsoft account, and your devices must meet the minimum [system requirements](https://account.microsoft.com/rewards/).
#### Redeeming your points
You may redeem your points, or you may contribute points to a listed nonprofit organization by visiting the [redeem page](https://aka.ms/redeemrewards) or by signing up for automatic contributions to non-profit organizations through a giving program.
#### Cancelling your participation
If you no longer want to participate in the Microsoft Rewards program, follow the instructions on the [opt out page](https://account.microsoft.com/rewards/optout?confirm=false). If you opt out, you will immediately lose all of your available points. 
See also:
* [Learn about Microsoft Rewards](https://support.microsoft.com/topic/learn-about-microsoft-rewards-c5ab735d-c6d9-4bb9-30ad-d828e954b6a9).
* The **Microsoft Rewards** section in the [Microsoft Services Agreement](https://www.microsoft.com/servicesagreement/).
```
* Please test the links.  The next-to-last link, I'm not able to follow the instruction " see the Rewards section at support.microsoft.com ("FAQ")", so changed it to:  `* [Learn about Microsoft Rewards](https://support.microsoft.com/topic/learn-about-microsoft-rewards-c5ab735d-c6d9-4bb9-30ad-d828e954b6a9).`
* Last link: removed /en-us per Docs Contributor Guide & doc build.
*

Reply:
applied locally

Reply:
A better version of the above Suggested Change has been incorped:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2574#rewards
* Also makes "How to earn" a heading.

@majenny you can Resolve this thread.  Distinct issue: you might want to test links at bottom of this h2 section, the links assume you're signed in, which I'm not (I should enter a comment to drive that).  Ihab helped set up my previous machine to partly use Rewards UI & a test account; that involved several locations, so I'm not attempting that level of testing on this machine.